rails/app/controllers/super_admin/users_controller.rb

```diff
@@ -0,0 +1,33 @@
+module SuperAdmin
+  class UsersController < ApplicationController
+    def edit
+      @user = User.find(params[:id])
+
+      if @user.role == 'admin'
```

Comment:
Since the roles are handled by an enum, we can use the built-in helper methods to check the role:

```suggestion
      if @user.admin?
```